src/plone/testing/z2.py

```diff
@@ -9,32 +6,32 @@
 
 
 deprecated(
-    'Please import from plone.testing.zope.',
-    Browser='plone.testing.zope:Browser',
-    TestIsolationBroken='plone.testing.zope:TestIsolationBroken',
-    installProduct='plone.testing.zope:installProduct',
-    uninstallProduct='plone.testing.zope:uninstallProduct',
-    login='plone.testing.zope:login',
-    logout='plone.testing.zope:logout',
-    setRoles='plone.testing.zope:setRoles',
-    makeTestRequest='plone.testing.zope:makeTestRequest',
-    addRequestContainer='plone.testing.zope:addRequestContainer',
-    zopeApp='plone.testing.zope:zopeApp',
-    Startup='plone.testing.zope:Startup',
-    STARTUP='plone.testing.zope:STARTUP',
-    IntegrationTesting='plone.testing.zope:IntegrationTesting',
-    INTEGRATION_TESTING='plone.testing.zope:INTEGRATION_TESTING',
-    FunctionalTesting='plone.testing.zope:FunctionalTesting',
-    FUNCTIONAL_TESTING='plone.testing.zope:FUNCTIONAL_TESTING',
-    ZServer='plone.testing.zope:WSGIServer',
-    ZSERVER_FIXTURE='plone.testing.zope:WSGI_SERVER_FIXTURE',
-    ZSERVER='plone.testing.zope:WSGI_SERVER',
+    "Please import from plone.testing.zope.",
+    Browser="plone.testing.zope:Browser",
+    TestIsolationBroken="plone.testing.zope:TestIsolationBroken",
+    installProduct="plone.testing.zope:installProduct",
+    uninstallProduct="plone.testing.zope:uninstallProduct",
+    login="plone.testing.zope:login",
+    logout="plone.testing.zope:logout",
+    setRoles="plone.testing.zope:setRoles",
+    makeTestRequest="plone.testing.zope:makeTestRequest",
+    addRequestContainer="plone.testing.zope:addRequestContainer",
+    zopeApp="plone.testing.zope:zopeApp",
+    Startup="plone.testing.zope:Startup",
+    STARTUP="plone.testing.zope:STARTUP",
+    IntegrationTesting="plone.testing.zope:IntegrationTesting",
+    INTEGRATION_TESTING="plone.testing.zope:INTEGRATION_TESTING",
+    FunctionalTesting="plone.testing.zope:FunctionalTesting",
+    FUNCTIONAL_TESTING="plone.testing.zope:FUNCTIONAL_TESTING",
+    ZServer="plone.testing.zope:WSGIServer",
+    ZSERVER_FIXTURE="plone.testing.zope:WSGI_SERVER_FIXTURE",
+    ZSERVER="plone.testing.zope:WSGI_SERVER",
 )
```

Comment:
Is there a plan to remove these BBB imports in the near future?

Reply:
I think we are already have deprecation warnings for them, if they state the time/release when that will happen, then yes, otherwise we can add them now and remove them by Plone 7.